### DISCUSSION TITLE
Short-term way to show hidden files for the current "open file" operation

### DISCUSSION THREAD

**Discussion by CBenoit** [helix-editor/helix#7712] [Ideas]
> Some short-term way to show hidden files for the current "open file" operation would be nice to have! Not having hidden files mess up the search result is the correct thing in 95% of the searches I do, so I am reluctant to change the default. It would be awfully convenient to be able to just toggle the "hidden" setting for the one search I am doing in the last 5%.

_Originally posted by @hunger in https://github.com/helix-editor/helix/discussions/1820#discussioncomment-2403515_

Just thought it would be useful to track this in a new issue.

**Comment by pascalkuthe** [comment]
You can do this with negative matches. For example `!/. !^.` hides all hidden files from the picker

**Reply by diegodorado** [reply]
negative matches may not cover this too well. Imagine you have lots of gitignore rules... you probably don't want to see those files in most cases ... but then you want to inspect, for example, the node_modules folder, and for that specific case you might want to toggle hidden paths

**Comment by diegodorado** [comment]
Would be nice to have a keybinding to toggle hidden files (besides the hidden config)

**Reply by gbogarinb** [reply]
I found this in https://github.com/helix-editor/helix/discussions/1360

```
[keys.normal."space"]
F = { a = ":toggle file-picker.git-ignore", F = "file_picker_in_current_directory"}
```
You can change the binding too if you prefer a different one.
